.github/workflows/append-release-notes.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Append to Release Notes
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v3.3.0
+        with:
+          repository: "trevorwhitney/grafana-github-actions"
```

Comment:
will update this to reference the upstream repo is a subsequent PR once the action is merged.